### DISCUSSION TITLE
How to store public key in library independent way

### DISCUSSION THREAD

**Discussion by hetin62** [Yubico/java-webauthn-server#254] [General]
Hi, 

How to store public key in library independent way?

does registrationResult.getPublicKeyCose() provide  library independent way?

Thanks

**Comment by emlun** [comment]
Hi! Yes, [`RegistrationResult.getPublicKeyCose()`](https://developers.yubico.com/java-webauthn-server/JavaDoc/webauthn-server-core/2.2.0/com/yubico/webauthn/RegistrationResult.html#getPublicKeyCose()) returns the [COSE-encoded credential public key](https://w3c.github.io/webauthn/#authdata-attestedcredentialdata-credentialpublickey). Any library that accepts COSE keys (including `java-webauthn-server`) should accept this value (once you unpack the raw `byte[]` array using `.getBytes()`.

Does that answer your question?
